### PR TITLE
Include OkHttp since customers typically use that; Finish the activity on Back Press

### PR DESCRIPTION
**Pull request by kvasilye**
*Description of changes:*

1 - Include OkHttp since customers typically use that
2 - Finish the activity on back press - Android 12 and newer will not do that, and you end up with the activity running in the background, annoying

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: BasicRealTime/build.gradle [hunk 1]
```diff
@@ -35,6 +35,10 @@ android {
 
 dependencies {
     implementation 'com.amazonaws:ivs-broadcast:1.12.1:stages@aar'
+
+    // The IVS SDK works without OkHttp, but customers typically use OkHttp so let's follow that
```
Comment (bclymer) [inline BasicRealTime/build.gradle:39]: Can we rephrase this to be more customer facing since this is our public repo? Maybe something like 
> The Amazon IVS SDK works without OkHttp, but because many applications include it, we are including it this demo app to mimic real world behavior.

Comment (kvasilye) [reply BasicRealTime/build.gradle]: Updated to use the exact wording you suggested

**Review by bclymer — APPROVED** [review]
